exercises/change/README.md

```diff
@@ -0,0 +1,44 @@
+# Change
+
+Correctly determine the change to be given using the least number of coins.
+
+The solution will need to accept a value of change to be given and an array of
+coin denominations.   The program returns the array of coin denominations to
+produce the correct amount of change.  For example, if change for 37 cents  
+is required from coins with the denominations of 1, 5, 10 and 25 then the
+result is an array with the values: 1, 1, 10 and 25.
+
+## Setup
+
+Go through the setup instructions for ECMAScript to
+install the necessary dependencies:
+
+http://exercism.io/languages/ecmascript
+
+## Requirements
+
+Install assignment dependencies:
+
+```bash
+$ npm install
+```
+
+## Making the Test Suite Pass
+
+Execute the tests with:
+
+```bash
+$ npm test
+```
+
+In many test suites all but the first test have been skipped.
```

Comment:
Probably should change this to read "In the test suite, all tests but the first have been skipped."  I'm not sure where this language came from, but I know it's present in many exercises.